### PR TITLE
Provide consistent examples of secrets management for all SDKs

### PR DESCRIPTION
**Pull request by amulyavarote**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Modified secrets management documentation page (under building blocks) for dapr.io

## Issue reference

This partially fulfills: #1969

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 1]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
...
+
+The below code examples loosely describe an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store secret in a local secret store.
+
+<img src="/images/building-block-secrets-management-example.png" width=1000 alt="Diagram showing secrets management of example service">
+
 ## Set up a secret store
 
 Before retrieving secrets in your application's code, you must have a secret store component configured. For the purposes of this guide, as an example you will configure a local secret store which uses a local JSON file to store secrets.
 
 >Note: The component used in this example is not secured and is not recommended for production deployments. You can find other alternatives [here]({{<ref supported-secret-stores >}}).
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
>Note: The component used in this example is not secure and is not recommended for production deployments. You can find other alternatives [here]({{<ref supported-secret-stores >}}).
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 2]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
   - name: secretsFile
-    value: <PATH TO SECRETS FILE>/mysecrets.json
+    value: secrets.json
   - name: nestedSeparator
     value: ":"
 ```
 
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
-
->Note: the path to the secret store JSON is relative to where you call `dapr run` from.
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md:45]: do not remove this note

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 3]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
 metadata:
-  name: my-secrets-store
+  name: localsecretstore
   namespace: default
 spec:
   type: secretstores.local.file
   version: v1
   metadata:
   - name: secretsFile
-    value: <PATH TO SECRETS FILE>/mysecrets.json
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md:38]: Add a comment that this is the path to secrets file.

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 4]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
 
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
-
->Note: the path to the secret store JSON is relative to where you call `dapr run` from.
-
-
 To configure a different kind of secret store see the guidance on [how to configure a secret store]({{<ref setup-secret-store>}}) and review [supported secret stores]({{<ref supported-secret-stores >}}) to see specific details required for different secret store solutions.
 ## Get a secret
 
 Now run the Dapr sidecar (with no application)
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]: This line conflicts with the examples shown as each runs dapr with an application.

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 5]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
+        static async Task Main(string[] args)
+        {
+            string SECRET_STORE_NAME = "localsecretstore";
+            using var client = new DaprClientBuilder().Build();
+            //Using Dapr SDK to get a secret
+            var secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result: {string.Join(", ", secret)}");
+            secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result for bulk: {string.Join(", ", secret.Keys)}");
+        }
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md:130]: What is the difference between line 131 and line 133 ?? both seem the same operation.

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 6]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
+
+	private static final String SECRET_STORE_NAME = "localsecretstore";
+
+	public static void main(String[] args) throws InterruptedException, JsonProcessingException {
+		DaprClient client = new DaprClientBuilder().build();
+        //Using Dapr SDK to get a secret
+		Map<String, String> secret = client.getSecret(SECRET_STORE_NAME, "secret").block();
+		log.info("Result: " + JSON_SERIALIZER.writeValueAsString(secret));
+		try {
+			secret = client.getSecret(SECRET_STORE_NAME, "secret").block();
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]: Both line 172 and 169 are the same.... 
can remove lines 169 and 170

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 7]
```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
...
+
+with DaprClient() as client:
+    #Using Dapr SDK to get a secret
+    secret = client.get_secret(store_name=DAPR_STORE_NAME, key=key)
+    logging.info('Result: ')
+    logging.info(secret.secret)
+    secret = client.get_bulk_secret(store_name=DAPR_STORE_NAME)
+    logging.info('Result for bulk secret: ')
+    logging.info(sorted(secret.secrets.items()))
+    try:
```
Comment (mukundansundar) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]: can remove the try catch block .... its the same in this example as lines 200-203

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 8]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
 
 This article provides guidance on using Dapr's secrets API in your code to leverage the [secrets store building block]({{<ref secrets-overview>}}). The secrets API allows you to easily retrieve secrets in your application code from a configured secret store.
 
+## Example
+
+The below code examples loosely describe an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store secret in a local secret store.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
The below code examples loosely describe an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store a secret in a local secret store.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 9]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
...
+
+<img src="/images/building-block-secrets-management-example.png" width=1000 alt="Diagram showing secrets management of example service">
+
 ## Set up a secret store
 
 Before retrieving secrets in your application's code, you must have a secret store component configured. For the purposes of this guide, as an example you will configure a local secret store which uses a local JSON file to store secrets.
 
->Note: The component used in this example is not secured and is not recommended for production deployments. You can find other alternatives [here]({{<ref supported-secret-stores >}}).
+>Note: The component used in this example is not secure and is not recommended for production deployments. You can find other alternatives [here]({{<ref supported-secret-stores >}}).
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
>Note: In a production-grade application, local secret stores are not recommended. You can find other alternatives [here]({{<ref supported-secret-stores >}}) to securely manage your secrets.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 10]
```diff
@@ -28,146 +34,242 @@ Create a directory for your components file named `components` and inside it cre
...
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
-
 >Note: the path to the secret store JSON is relative to where you call `dapr run` from.
 
-
 To configure a different kind of secret store see the guidance on [how to configure a secret store]({{<ref setup-secret-store>}}) and review [supported secret stores]({{<ref supported-secret-stores >}}) to see specific details required for different secret store solutions.
 ## Get a secret
 
-Now run the Dapr sidecar (with no application)
+Now run the Dapr sidecar with the application.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
Run the Dapr sidecar with the application.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 11]
```diff
@@ -28,146 +34,242 @@ Create a directory for your components file named `components` and inside it cre
...
-func main() {
-  url := "http://localhost:3500/v1.0/secrets/my-secrets-store/my-secret"
+{{< /tabs >}}
 
-  res, err := http.Get(url)
-  if err != nil {
-    panic(err)
-  }
-  defer res.Body.Close()
+And now you can get the secret by calling the Dapr sidecar using the secrets API:
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
Get the secret by calling the Dapr sidecar using the secrets API:
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 12]
```diff
@@ -28,146 +34,242 @@ Create a directory for your components file named `components` and inside it cre
...
-        .then((response) => {
-            if (!response.ok) {
-                throw "Could not get secret";
-            }
-            return response.text();
-        }).then((secret) => {
-            console.log(secret);
-        });
-```
+Once you have a secret store set up, call Dapr to get the secrets from your application code. Below are code examples that leverage Dapr SDKs for retrieving a secret.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
Once you have a secret store, call Dapr to get the secrets from your application code. Below are code examples that leverage Dapr SDKs for retrieving a secret.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 13]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
 
 This article provides guidance on using Dapr's secrets API in your code to leverage the [secrets store building block]({{<ref secrets-overview>}}). The secrets API allows you to easily retrieve secrets in your application code from a configured secret store.
 
+## Example
+
+The below code examples loosely describe an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store a secret in a local secret store.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
The below code examples loosely describes an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store a secret in a local secret store.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 14]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
 
 This article provides guidance on using Dapr's secrets API in your code to leverage the [secrets store building block]({{<ref secrets-overview>}}). The secrets API allows you to easily retrieve secrets in your application code from a configured secret store.
 
+## Example
+
+The below code examples loosely describes an application that processes orders. In the examples, there is an order processing service which has a Dapr sidecar. The order processing service uses Dapr to store a secret in a local secret store.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
The below code example loosely describes an application that processes orders. In the example, there is an order processing service, which has a Dapr sidecar. The order processing service uses Dapr to store a secret in a local secret store.
```

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 15]
```diff
@@ -8,17 +8,23 @@ description: "Use the secret store building block to securely retrieve a secret"
...
+
+<img src="/images/building-block-secrets-management-example.png" width=1000 alt="Diagram showing secrets management of example service">
+
 ## Set up a secret store
 
 Before retrieving secrets in your application's code, you must have a secret store component configured. For the purposes of this guide, as an example you will configure a local secret store which uses a local JSON file to store secrets.
 
->Note: The component used in this example is not secured and is not recommended for production deployments. You can find other alternatives [here]({{<ref supported-secret-stores >}}).
+>Note: In a production-grade application, local secret stores are not recommended. You can find other alternatives [here]({{<ref supported-secret-stores >}}) to securely manage your secrets.
```
Comment (greenie-msft) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
{{% alert title="Warning" color="warning" %}}
In a production-grade application, local secret stores are not recommended. You can find other alternatives [here]({{<ref supported-secret-stores >}}) to securely manage your secrets.
{{% /alert %}}
```

**Review by greenie-msft — APPROVED** [review]

**Review by willtsai — APPROVED** [review]

**Pull request merged**